doc/reference/reference_lua/box_stat/memtx.rst

```diff
@@ -0,0 +1,81 @@
+.. _box_introspection-box_stat_memtx:
+
+box.stat.memtx()
+================
+
+.. module:: box.stat
+
+.. function:: memtx()
+
+    Shows memtx-storage-engine activity.
+
+.. _box_introspection-box_stat_memtx_tx:
+
+box.stat.memtx().tx
+-------------------
+
+* ``tx`` shows the statistics of the memtx transactional manager,
+  which is responsible for transactions (``box.stat.memtx().tx.txn``)
+  and multiversion concurrency control (``box.stat.memtx().tx.mvcc``).
+
+  * ``box.stat.memtx().tx.txn`` shows memory allocation related to transactions.
+
+    It consists of the following sections:
+
+    * ``statements`` are *transaction statements*.
+      As an example, consider a user starting a transaction with
+      ``space:replace{0, 1}`` within this transaction. Under the hood,
+      this operation becomes a statement for this transaction.
+    * ``user`` is the memory that a user allocated within
+      the current transaction using the Tarantool C API function
+      :ref:`box_txn_alloc() <txn-box_txn_alloc>`.
+    * ``system`` is the memory allocated for internal needs
+      (for example, logs) and savepoints.
+
+      For each section, Tarantool reports the following statistics:
+
+      * ``total`` is the number of bytes that are currently allocated in memtx
+        for all transactions within the section scope.
+      * ``avg`` is the average number of bytes that a single transaction uses
+        (equals ``total`` / number of open transactions).
+      * ``max`` is the maximal number of bytes that a single transaction uses.
```

Comment:
```suggestion
      * ``max`` is the maximum number of bytes that a single transaction uses.
```

Sounds more correct to me. Like in ``maximum speed``, ``maximum value``. I checked the difference here ["Maximum" vs. "maximal"](https://english.stackexchange.com/questions/38260/maximum-vs-maximal).

Reply:
I see the reaction (👍 ) but not the change. Do you agree?)
<img width="510" alt="image" src="https://github.com/user-attachments/assets/3e11d47d-fde0-4436-bb3c-dc12983ba055">